tests/components/loqed/test_sensor.py

```diff
@@ -33,38 +28,6 @@ async def test_battery_sensor(
     assert state.attributes[ATTR_STATE_CLASS] is SensorStateClass.MEASUREMENT
 
 
-async def test_wifi_stength_sensor(
```

Comment:
You completely removed the test for this sensor because its disabled by default. There is actually a little trick to enable the sensor in the test and test its state after that. See zwave_js for an example: https://github.com/home-assistant/core/blob/dev/tests/components/zwave_js/test_sensor.py#L180